huxley/core/models.py

```diff
@@ -383,10 +509,13 @@ class Meta:
         db_table = u'assignment'
         unique_together = ('committee', 'country')
 
+
 pre_save.connect(Assignment.update_assignment, sender=Assignment)
 
+
 class CountryPreference(models.Model):
     school = models.ForeignKey(School)
```

Comment:
We can keep the school for now, but we should plan to access the school via the registration. Otherwise the information is redundant. This can be done in followup PRs; I'm wary of one PR getting too big.